### PR TITLE
Remove apipie from properties controller

### PR DESCRIPTION
**Pull request by kmahern**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-3161)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: spec/requests/properties_controller_spec.rb [hunk 1]
```diff
@@ -61,6 +61,129 @@
...
+                  value: 1000,
+                  outstanding_mortgage: 0,
+                  shared_with_housing_assoc: false,
+                },
+              ],
+            },
+          }
+        end
+
+        it "returns expected error response", :show_in_doc do
```
Comment (jsugarman) [inline spec/requests/properties_controller_spec.rb:146]: `:show_in_doc` is an api pie helper i believe

File: spec/requests/properties_controller_spec.rb [hunk 2]
```diff
@@ -49,7 +49,7 @@
         end
       end
 
-      context "Invalid assessment ID causes service to return failure" do
+      context "with invalid assessment ID" do
         let(:assessment_id) { SecureRandom.uuid }
 
         it "returns expected error response", :show_in_doc do
```
Comment (jsugarman) [inline spec/requests/properties_controller_spec.rb:55]: :show_in_doc is an api pie helper i believe

File: spec/requests/properties_controller_spec.rb [hunk 3]
```diff
@@ -61,6 +61,129 @@
           expect(response).to have_http_status(:unprocessable_entity)
         end
       end
+
+      context "with missing main_home attribute" do
+        let(:request_payload) { { properties: {} } }
+
+        it "returns expected error response", :show_in_doc do
```
Comment (jsugarman) [inline spec/requests/properties_controller_spec.rb:68]: :show_in_doc is an api pie helper i believe

File: spec/requests/properties_controller_spec.rb [hunk 4]
```diff
@@ -61,6 +61,129 @@
...
+                value: "one hundred pounds",
+                outstanding_mortgage: 200,
+                percentage_owned: 15,
+                shared_with_housing_assoc: true,
+              },
+            },
+          }
+        end
+
+        it "returns expected error response", :show_in_doc do
```
Comment (jsugarman) [inline spec/requests/properties_controller_spec.rb:92]: :show_in_doc is an api pie helper i believe

File: spec/requests/properties_controller_spec.rb [hunk 5]
```diff
@@ -61,6 +61,129 @@
...
+              main_home: {
+                value: 500_000,
+                percentage_owned: 15,
+                shared_with_housing_assoc: true,
+              },
+            },
+          }
+        end
+
+        it "returns expected error response", :show_in_doc do
```
Comment (jsugarman) [inline spec/requests/properties_controller_spec.rb:115]: :show_in_doc is an api pie helper i believe

File: spec/requests/properties_controller_spec.rb [hunk 6]
```diff
@@ -61,6 +61,129 @@
...
+                  outstanding_mortgage: 0,
+                  percentage_owned: 99,
+                  shared_with_housing_assoc: "false",
+                },
+              ],
+            },
+          }
+        end
+
+        it "returns expected error response", :show_in_doc do
```
[the remark this answers is not in the document]
Comment (kmahern) [reply spec/requests/properties_controller_spec.rb]: 👍 thanks, have removed!

File: spec/requests/properties_controller_spec.rb (hunk 2 above)
Comment (kmahern) [reply spec/requests/properties_controller_spec.rb]: removed

File: spec/requests/properties_controller_spec.rb (hunk 3 above)
Comment (kmahern) [reply spec/requests/properties_controller_spec.rb]: removed

File: public/schemas/properties.json.erb [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+      "properties": {
+        "main_home": {
+          "type": "object",
+          "required": ["value", "outstanding_mortgage", "percentage_owned", "shared_with_housing_assoc"],
+          "properties": {
+            "value": {
+              "$ref": "<%= "file://#{@schema_dir}/common.json#currency" %>"
+            },
+            "outstanding_mortgage": {
+              "$ref": "<%= "file://#{@schema_dir}/common.json#currency" %>"
```
Comment (jsugarman) [inline public/schemas/properties.json.erb:20]: should be positive_currency, when comparing to apipie

**Review by jsugarman — APPROVED** [review]
👍 approved

**Comment by jsugarman** [comment]
as discussed regarding amendment/checking of `apipie_examples.json` inline with these changes/removals

**Pull request merged**